app/routers/event.py

```diff
@@ -149,7 +149,8 @@ def is_fields_types_valid(to_check: Dict[str, Any], types: Dict[str, Any]):
     """validate dictionary values by dictionary of types"""
     errors = []
     for field_name, field_type in to_check.items():
-        if types[field_name] and not isinstance(field_type, types[field_name]):
+        if types[field_name] \
+                and not isinstance(field_type, (types[field_name])):
```

Comment:
The parenthesis around `types..` are unnecessary.

Reply:
No , without ( ) I get an exception "isinstance should get Tuple "